verilog/CST/net.cc

```diff
@@ -29,4 +33,13 @@ std::vector<verible::TreeSearchMatch> FindAllNetDeclarations(
   return SearchSyntaxTree(root, NodekNetDeclaration());
 }
 
+const SyntaxTreeLeaf* GetIdentifierFromNetDeclaration(
+    const Symbol& symbol) {
+  const auto* data_type =
+      verible::GetSubtreeAsSymbol(symbol, NodeEnum::kNetDeclaration, 1);
+  const auto* identifier_symbol =
+      verible::GetSubtreeAsSymbol(*data_type, NodeEnum::kDataTypeImplicitIdDimensions, 1);
+  return AutoUnwrapIdentifier(*ABSL_DIE_IF_NULL(identifier_symbol));
```

Comment:
I think the grammar guarantees that these will be simple identifiers (never qualified), so the AutoUnwrap is should be unnecessary (but still safe).